b2/b2.py

```diff
@@ -808,6 +808,34 @@ def list_file_versions(self, start_filename=None, start_file_id=None, max_entrie
         }
         return post_json(url, params, auth_token)
 
+    def list_unfinished_large_files(self, start_file_id=None, batch_size=None):
+        """
+        A generator that yields an UnfinishedLargeFile for each
+        unfinished large file in the bucket, starting at the
+        given file.
+        """
+        account_info = self.api.account_info
+        batch_size = batch_size or 100
+        while True:
+            batch = self.api.raw_api.list_unfinished_large_files(
+                account_info.get_api_url(), account_info.get_account_auth_token(), self.id_,
+                start_file_id, batch_size
+            )
+            for file_dict in batch['files']:
+                yield UnfinishedLargeFile(file_dict)
+            start_file_id = batch['nextFileId']
```

Comment:
`start_file_id = batch.get('nextFileId')` so that `None` is returned when the key is missing, as expected in the following code, instead of `KeyError`